addon/components/drag-sort-item.js

```diff
@@ -28,6 +28,7 @@ export default Component.extend({
   childTagName    : 'div',
   dragEndAction   : undefined,
   draggingEnabled : undefined,
+  disableSorting  : undefined,
```

Comment:
The same line should be added to `components/drag-sort-list.js` to indicate that it's one of possible params.